c10/core/op_schemas/layer_norm.cpp

```diff
@@ -0,0 +1,4 @@
+#include <c10/core/op_schemas/layer_norm.h>
+#include <c10/core/dispatch/OpSchemaRegistration.h>
+
+C10_DEFINE_OP_SCHEMA(c10::core::ops::LayerNorm);
```

Comment:
Nit: if you're going to put this in the `ops` namespace, can you make the directory reflect this too?

Reply:
Named both namespace and directory `opschema`